src/ontodia/data/sparql/provider.ts

```diff
@@ -77,7 +79,27 @@ export class SparqlDataProvider implements DataProvider {
             }}
         `;
         return executeSparqlQuery<Sparql.ElementsInfoResponse>(this.endpointUrl, query).
-            then(elementsInfo => getElementsInfo(elementsInfo, params.elementIds));
```

Comment:
Wrap the dot (`.`) in `.then()` call to the next line to better distinguish it from standalone function calls.